pl_examples/basic_examples/mnist_examples/README.md

```diff
@@ -0,0 +1,67 @@
+## MNIST Examples
+
+Here are 5 MNIST examples showing you how to gradually convert from pure PyTorch to PyTorch Lightning.
+
+The transition through [LightningLite](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.rst) from pure PyTorch is optional but it might be helpful to learn about it.
+
+#### 1. Image Classifier with Vanilla PyTorch
+
+Trains a simple CNN over MNIST using vanilla PyTorch.
+
+```bash
+# cpu
+python image_classifier_1_pytorch.py
+```
```

Comment:
can we rather include the code as @ethanwharris does in Flash?

Reply:
Like what exactly? There is no readme file in flash: https://github.com/PyTorchLightning/lightning-flash/tree/master/flash_examples